eva_submission/eload_validation.py

```diff
@@ -304,7 +314,10 @@ def report(self):
         }
 
         report = """Validation performed on {validation_date}
-eva-sub-cli: {eva_sub_cli}
+VCF checks: {vcf_check}
+Assembly checks: {assembly_check}
+Metadata check: {metadata_check}
+Sample check: {sample_check}
```

Comment:
Is it worth presenting this as sub tasks of eva-sub-cli  like
```suggestion
eva-sub-cli: 
  VCF checks: {vcf_check}
  Assembly checks: {assembly_check}
  Metadata check: {metadata_check}
  Sample check: {sample_check}
```